beacon-chain/orchestrator/client_test.go

```diff
@@ -0,0 +1,7 @@
+package orchestrator
+
+import "testing"
+
+func TestRPCClient_ConfirmVanBlockHashes(t *testing.T) {
+
```

Comment:
wiggle

Reply:
tests have been refactored to proper one